src/net/sourceforge/kolmafia/textui/DataTypes.java

```diff
@@ -1065,6 +1096,11 @@ private static String promptForValue(final Type type, final String message, fina
       case STAT:
         return InputFieldUtilities.input(message, DataTypes.STAT_ARRAY);
 
+      case MODIFIER:
+        return InputFieldUtilities.input(
+                message, ModifierDatabase.allModifiers().toArray(new Modifier[] {}))
```

Comment:
This will be big. Is it sorted?
As seen above, no.
Should it be?

Reply:
it's sorted by type (numeric, then string, then boolean), but inside the types it's "sorted" by ordering within the enums.

Ordering by name inside the types would make more sense. Alternatively, they compare as strings (`isStringLike`), so perhaps they should be sorted by name as well.

Alternatively, it doesn't matter: I can't see any use cases that would care about the ordering.

Reply:
This particular piece of code is passing an array to be used in a combo box in a dialog. Which is to say, the user will scroll through it looking for the item to select.

Hence my thinking pure alphabetical would be the best for this user-facing use case.

Reply:
I think this is reasonable, and I think pure alphabetical is also reasonable in general for consistency.